pom.xml

```diff
@@ -2,64 +2,91 @@
+            <artifactid>slf4j-simple</artifactid>
             <version>1.7.36</version>
         </dependency>
         <dependency>
-            <groupId>com.sparkjava</groupId>
-            <artifactId>spark-core</artifactId>
+            <groupid>com.sparkjava</groupid>
+            <artifactid>spark-core</artifactid>
             <version>2.9.3</version>
         </dependency>
         <dependency>
-            <groupId>org.json</groupId>
-            <artifactId>json</artifactId>
+            <groupid>org.json</groupid>
+            <artifactid>json</artifactid>
             <version>20211205</version>
         </dependency>
         <dependency>
-            <groupId>org.jgrapht</groupId>
-            <artifactId>jgrapht-core</artifactId>
+            <groupid>org.jgrapht</groupid>
+            <artifactid>jgrapht-core</artifactid>
             <version>1.5.1</version>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupid>junit</groupid>
+            <artifactid>junit</artifactid>
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
 
-    <packaging>jar</packaging>
-
     <build>
         <plugins>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>1.4.0</version>
+                <groupid>org.codehaus.mojo</groupid>
+                <artifactid>exec-maven-plugin</artifactid>
+                <version>3.0.0</version>
+                <configuration>
+                    <mainclass>swapsolver</mainclass>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupid>org.apache.maven.plugins</groupid>
+                <artifactid>maven-compiler-plugin</artifactid>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                    <encoding>${project.encoding}</encoding>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupid>org.apache.maven.plugins</groupid>
+                <artifactid>maven-checkstyle-plugin</artifactid>
+                <version>3.1.2</version>
                 <configuration>
-                    <mainClass>SwapSolver</mainClass>
+                    <encoding>${project.encoding}</encoding>
+                    <configlocation>.checkstyle.xml</configlocation>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupid>net.revelc.code.formatter</groupid>
+                <artifactid>formatter-maven-plugin</artifactid>
+                <version>2.10.0</version>
+                <configuration>
+                    <encoding>${project.encoding}</encoding>
```

Comment:
Add
```
 <excludes>
      <exclude>pom.xml</exclude>
</excludes>
```
to exclude pom.xml ([Source](https://code.revelc.net/formatter-maven-plugin/examples.html))

Reply:
I must have formatted with another tool. It is fixed now 💪